scan.sh

```diff
@@ -7,6 +7,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+install_image_scan() {
+    local cs_py="cs_scanimage.py"
+    if [[ ! -f "$GITHUB_ACTION_PATH/$cs_py" ]]; then
+        echo "Installing container-image-scan..."
+        local latest_pkg=$(curl -s https://api.github.com/repos/crowdstrike/container-image-scan/releases/latest | jq -r '.assets[0].browser_download_url')
```

Comment:
@isimluk maybe you can recommend the most secure way to just pipe this to bash ;)

Reply:
🙊